README.md

```diff
@@ -103,7 +103,7 @@ cd sushi-fusion
 npm i && npm start
 ```
 
-Then open [http://plaza-athenee.surge.sh/](http://plaza-athenee.surge.sh/) to see the app.
+Then open [http://localhost:3000](http://plaza-athenee.surge.sh/) to see the app.
```

Comment:
```suggestion
Then open `http://localhost:3000/` to see the app.
```

Reply:
The task was when I will click on `http://localhost:3000/` this link it will open in a new tab and visit that link.
Is that correct?

Reply:
NO, I think that was a misunderstanding, the idea was that the link was not working. Still, in this case, the link is not "fixable" because the idea of the text was something on the line of "what to do when you start the development app", I think only saying what the URL is is enough even if is not in a link form.